Lib/pathlib.py

```diff
@@ -1284,6 +1287,14 @@ def rmdir(self):
             self._raise_closed()
         self._accessor.rmdir(self)
 
+    def rmtree(self, ignore_errors=False, onerror=None):
+        """
+        Remove this directory tree.
```

Comment:
Use a line for docstring

Reply:
I do no understand what you mean?

I've tried to format my docstring in the same way as other methods:

```python
    def rmdir(self):
        """
        Remove this directory.  The directory must be empty.
        """
```